scalafx/src/main/scala/scalafx/scene/control/MenuButton.scala

```diff
@@ -103,4 +103,49 @@ class MenuButton(override val delegate: jfxsc.MenuButton = new jfxsc.MenuButton)
   def show() {
     delegate.show()
   }
+
+  /**
+    * Called just prior to the `ContextMenu` being hidden.
+    *
+    * @return the on hiding property
+    * @since 10
+    */
+  def onHiding = delegate.onHidingProperty
```

Comment:
Should provide explicit return type: `ObjectProperty[EventHandler[Event]]`
Fully expanded it would be: `scalafx.beans.property.ObjectProperty[javafx.event.EventHandler[javafx.event.Event]]`